pkgs/applications/misc/hyper/default.nix

```diff
@@ -11,11 +11,11 @@ let
   ];
 in
 stdenv.mkDerivation rec {
-  version = "0.7.6";
-  name = "hyperterm-${version}";
+  version = "0.8.3";
+  name = "hyper-${version}";
   src = fetchurl {
```

Comment:
Seems like a good idea to add `hyperterm = self.hyper`, would that go in `pkgs/top-level/all-packages.nix`?